tests/drawing/test_layout.py

```diff
@@ -135,3 +135,67 @@ def test_pca_transform():
     assert np.allclose(transform1_pos2[1], np.array([-0.67296626, -0.21706316]))
     assert np.allclose(transform1_pos2[2], np.array([-0.02177678, -2.23596193]))
     assert np.allclose(transform1_pos2[3], np.array([4.47192387, -0.04355357]))
+
+def test_circular_layout():
+    # hypergraph
+    H = xgi.random_hypergraph(10, [0.2], seed=1)
+    pos = xgi.circular_layout(H)
+    assert len(pos) == H.num_nodes
+    
+    # empty hypergraph
+    H = xgi.empty_hypergraph()
+    pos = xgi.circular_layout(H)
+    assert pos == {}
+    
+    # single node hypergraph
+    H = xgi.Hypergraph()
+    H.add_node(1)
```

Comment:
Now you can use this 
```suggestion
    H = xgi.trivial_hypergraph()
```